doc/source/guide/data_types/maps.rst

```diff
@@ -57,59 +53,63 @@ The format of the header follows the FITS standard.
 
```

Comment:
Should probably add in something about masked numpy arrays. Maybe something like "SunPy maps accept numpy masked arrays as data input".